datalad/interface/run.py

```diff
@@ -113,6 +116,13 @@ class Run(Interface):
             commit message.""",
             constraints=EnsureNone() | EnsureChoice("inputs", "outputs", "both")),
         message=save_message_opt,
+        sidecar=Parameter(
+            args=('--sidecar',),
+            metavar="yes|no",
+            doc="""If given, a record with information on the command execution
+            is not placed into the commit message, but into a separate record
+            file instead.""",
+            constraints=EnsureNone() | EnsureBool()),
```

Comment:
Should be `EnsureChoice` instead of `EnsureBool`, right?

Reply:
No, that is not needed. If given a non-bool type, it will honor the following mapping:

```
    False: '0', 'no', 'off', 'disable', 'false'
    True: '1', 'yes', 'on', 'enable', 'true'
 ```